scripts/rmc.py

```diff
@@ -93,19 +93,86 @@ def symbol_table_to_gotoc(json_filename, cbmc_filename, verbose=False, keep_temp
     return process.returncode
 
 
-def run_cbmc(cbmc_filename, cbmc_args, verbose=False, quiet=False):
+def run_cbmc(cbmc_filename, cbmc_args, verbose=False, quiet=False, output_to=None):
```

Comment:
I don't understand why now we want to write output to a file and not before.

Reply:
Before, `run_cbmc` always just prints to stdout (assuming `quiet` is False), but we need to redirect it to `result.xml`, etc.

Reply:
Fixed, no longer needed.